### ISSUE TITLE
Wheel request: Thrustmaster T598

### ISSUE THREAD

**Issue by Thalley** [berarma/oversteer#241]
The T598 (https://www.thrustmaster.com/products/t598/) is a recent addition to the Thrustmaster lineup. The wheel works for both PC and Playstation, and is a entry-level wheel with direct (axial) drive

**Comment by berarma** [comment]
I would need the product id as shown by `lsusb`, the axis and buttons events as shown by `evtest` and the max rotation range.

**Comment by Thalley** [comment]
> I would need the product id as shown by `lsusb`, the axis and buttons events as shown by `evtest` and the max rotation range.

Makes sense. I'll try to remember to post it here once I get one myself :)
